PictureViewMaster/PictureMasterView/PictureMasterImageView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  PictureMasterImageView.swift
+//  PictureViewMaster
+//
+//  Created by El gera de la gente on 4/6/16.
+//  Copyright © 2016 Inaka. All rights reserved.
+//
+
+import UIKit
+
+@objc protocol MasterViewDelegate: NSObjectProtocol {
+    func showImageInMasterView (image: UIImage)
```

Comment:
To follow normal conventions when developing delegates, a better fit would be `func pictureMasterImageViewDidReceiveTap(imageView: PictureMasterImageView)` where you pass in `self` when calling it, and then you'd use `imageView.image` to retrieve the image in the class implementing this delegate.